src/js/views/Search.js

```diff
@@ -75,6 +75,7 @@ const Search = () => {
     { value: 'followers', label: i18n('common.popularity') },
     { value: 'artist', label: i18n('common.artist') },
     { value: 'duration', label: i18n('common.duration') },
+    { value: 'asloaded', label: i18n('search.asloaded')}

```

Comment:
Have a gander at the equivalent syntax for other views, for instance `src/js/views/Library/Albums.jsx`.

"As loaded" translates quite literally to "don't sort me!" ie `sortOrder: null`:

```
      {
        value: null,
        label: i18n('fields.filters.as_loaded'),
      }
```

Reply:
Yes, that seems like a better way to do it!